biome.json

```diff
@@ -1,7 +1,7 @@
 {
   "formatter": {
     "indentStyle": "space",
-    "indentSize": 2,
+    "indentWidth": 2,
```

Comment:
Hmm, this shouldn't be causing a linter issue, but worth doing anyhow. I'll do that in a separate commit.

Reply:
It threw an error before, so I thought I'd try to fix it

Reply:
Should I undo this?

Reply:
Actually, it looks like we can't switch to this yet, Biome is still in the process of switching.

Reply:
Hmm, okay, I've made `indentWidth` work now.